bears/general/InvalidLinkBear.py

```diff
@@ -26,8 +29,16 @@ def get_status_code_or_error(url, timeout):
         except requests.exceptions.RequestException:
             pass
 
-    @staticmethod
-    def find_links_in_file(file, timeout):
+    def get_status_code_checked(self, url, timeout):
```

Comment:
i'm sorry if i missed something but i'm not sure why we need this function?, can't we have a `check_connection` param in the `get_status_code`  function and do the check there itself?